src/types/models/index.ts

```diff
@@ -137,6 +137,7 @@ export interface SavedSearch {
   language: string
   searchQuery: ListingFilterParams
   uiMetadata: Record<string, unknown>
+  enabledUntil?: string
```

Comment:
Will there be saved searches not expiring?

Reply:
I fixed that ! Thank you Lorenzo !

Reply:
Actually, I think it should remain optional. When the savedSearch is sent to BE the first time, we don't want to specify the expiration date since this data will be generated by BE. I noticed this when I tried this new version on my branch, I have a type error stating that `enabledUntil` is missing[ here ](https://github.com/carforyou/carforyou-listings-web/blob/678b653ea9ff023ace2ba6e73715f69d1ac21f5b/components/forms/savedSearchForm.tsx#L57)

Reply:
We discussed in a call: it is only not present when creating a saved search so we decided to keep it mandatory as part of the type and omit it for creation.